careerfairsystems/arkad-app-2018
# Add search field on companies list

## Change description

Adds a search field on the company list screen that can be used to search company names.

## How to verify

Run the project and navigate to the Companies tab. Confirm that the search field is displayed and that it works as expected.

## Issues fixed

This fixes #43.
